Assets/Tests/InputSystem/Plugins/UITests.cs

```diff
@@ -3567,22 +3568,13 @@ public void UI_CanDriveVirtualMouseCursorFromGamepad()
     // to our manifest without breaking test runs with previous versions of Unity. However, in 2021.2, all the UITK functionality
     // has moved into the com.unity.modules.uielements module which is also available in previous versions of Unity. This way we
     // can have a reference to UITK that doesn't break things in previous versions of Unity.
-#if UNITY_2021_2_OR_NEWER
+#if UNITY_2022_3_OR_NEWER
     [UnityTest]
     [Category("UI")]
-    [TestCase(UIPointerBehavior.AllPointersAsIs, ExpectedResult = 1
-#if TEMP_DISABLE_UITOOLKIT_TEST && (UNITY_STANDALONE_OSX || UNITY_STANDALONE_WIN)
-        , Ignore = "Currently fails on MacOS, MacOS standalone, MacOS standalone IL2CPP player on Unity version 2022.2 CI"
-#endif
-     )]
-    [TestCase(UIPointerBehavior.SingleMouseOrPenButMultiTouchAndTrack, ExpectedResult = 1
-#if TEMP_DISABLE_UITOOLKIT_TEST && (UNITY_STANDALONE_OSX)
-            // temporarily disable this test case on OSX player for 2021.2. It only intermittently works and I don't know why!
-        , Ignore = "Currently fails on OSX IL2CPP player on Unity version 2021.2"
-#endif
-     )]
+    [TestCase(UIPointerBehavior.AllPointersAsIs, ExpectedResult = 1)]
+    [TestCase(UIPointerBehavior.SingleMouseOrPenButMultiTouchAndTrack, ExpectedResult = 1)]
     [TestCase(UIPointerBehavior.SingleUnifiedPointer, ExpectedResult = 1)]
-#if (UNITY_ANDROID || UNITY_IOS || UNITY_TVOS) || (TEMP_DISABLE_UITOOLKIT_TEST && (UNITY_STANDALONE_OSX || UNITY_STANDALONE_WIN))
+#if UNITY_ANDROID || UNITY_IOS || UNITY_TVOS
     [Ignore("Currently fails on the farm but succeeds locally on Note 10+; needs looking into.")]
```

Comment:
does it still fail on the farm after your changes?

Reply:
I didn't see any fail at all in the current state of the branch. All the tests that are failing are also failing on `develop`.